app/src/org/commcare/android/database/connect/models/ConnectJobRecordV5.java

```diff
@@ -0,0 +1,149 @@
+    public static final String META_BUDGET_TOTAL = "total_budget";
+    public static final String META_LAST_WORKED_DATE = "last_worked";
+    public static final String META_STATUS = "status";
+    public static final String META_LEARN_MODULES = "total_modules";
+    public static final String META_COMPLETED_MODULES = "completed_modules";
+
+    public static final String META_LEARN_PROGRESS = "learn_progress";
+    public static final String META_DELIVERY_PROGRESS = "deliver_progress";
+    public static final String META_LEARN_APP = "learn_app";
+    public static final String META_DELIVER_APP = "deliver_app";
+    public static final String META_CLAIM = "claim";
+    public static final String META_CLAIM_DATE = "date_claimed";
+    public static final String META_MAX_PAYMENTS = "max_payments";
+    public static final String META_CURRENCY = "currency";
+    public static final String META_ACCRUED = "payment_accrued";
+    public static final String META_SHORT_DESCRIPTION = "short_description";
+
+    public static final String META_USER_SUSPENDED = "is_user_suspended";
+
+
+    @Persisting(1)
+    @MetaField(META_JOB_ID)
+    private int jobId;
+    @Persisting(2)
+    @MetaField(META_NAME)
+    private String title;
+    @Persisting(3)
+    @MetaField(META_DESCRIPTION)
+    private String description;
+    @Persisting(4)
+    @MetaField(META_ORGANIZATION)
+    private String organization;
+    @Persisting(5)
+    @MetaField(META_END_DATE)
+    private Date projectEndDate;
+    @Persisting(6)
+    @MetaField(META_BUDGET_PER_VISIT)
+    private int budgetPerVisit;
+    @Persisting(7)
+    @MetaField(META_BUDGET_TOTAL)
+    private int totalBudget;
+    @Persisting(8)
+    @MetaField(META_MAX_VISITS)
+    private int maxVisits;
+    @Persisting(9)
+    @MetaField(META_MAX_DAILY_VISITS)
+    private int maxDailyVisits;
+    @Persisting(10)
+    @MetaField(META_DELIVERY_PROGRESS)
+    private int completedVisits;
+    @Persisting(11)
+    @MetaField(META_LAST_WORKED_DATE)
+    private Date lastWorkedDate;
+    @Persisting(12)
+    @MetaField(META_STATUS)
+    private int status;
+    @Persisting(13)
+    @MetaField(META_LEARN_MODULES)
+    private int numLearningModules;
+    @Persisting(14)
+    @MetaField(META_COMPLETED_MODULES)
+    private int learningModulesCompleted;
+    @Persisting(15)
+    @MetaField(META_CURRENCY)
+    private String currency;
+    @Persisting(16)
+    @MetaField(META_ACCRUED)
+    private String paymentAccrued;
+    @Persisting(17)
+    @MetaField(META_SHORT_DESCRIPTION)
+    private String shortDescription;
+    @Persisting(18)
+    private Date lastUpdate;
+    @Persisting(19)
+    private Date lastLearnUpdate;
+    @Persisting(20)
+    private Date lastDeliveryUpdate;
+    @Persisting(21)
+    @MetaField(META_CLAIM_DATE)
+    private Date dateClaimed;
+
+    @Persisting(22)
+    @MetaField(META_USER_SUSPENDED)
+    private boolean isUserSuspended;
```

Comment:
Oops! V5 still needs to have the dateClaimed (22) and projectStartDate (23) fields, but shouldn't have any knowledge about isUserSuspended (24).

Essentially, this class needs to match the exact structure of the DB before we perform the upgrade.